datafusion/core/tests/sql/group_by.rs

```diff
@@ -18,6 +18,8 @@
 use super::*;
 
 #[tokio::test]
+#[ignore]
+// TODO: issue: https://github.com/apache/arrow-datafusion/issues/6623
```

Comment:
Here is a proposed fix: https://github.com/apache/arrow-datafusion/pull/6632

Reply:
I think we have fixed the date_trunc issues now, so these workarounds should no longer be necessary